packages/attribute-registry-process/test/vitest.api.setup.ts

```diff
@@ -28,6 +28,7 @@ vi.mock("pagopa-interop-commons", async () => {
             const jwtToken = jwtFromAuthHeader(req, genericLogger);
             const decoded = decodeJwtToken(jwtToken, genericLogger);
             const ctx = req.ctx || {};
+            // eslint-disable-next-line functional/immutable-data
```

Comment:
why this change? this should already be disabled by `packages/attribute-registry-process/test/.eslintrc.json`

Reply:
I reverted this change in b5b28b4a28f3c87217250416c97e3e22f77e74c5